PWGLF/TableProducer/LFTreeCreatorNuclei.cxx

```diff
@@ -52,6 +52,7 @@ DECLARE_SOA_COLUMN(Pz, pz, float);
 DECLARE_SOA_COLUMN(Pt, pt, float);
 DECLARE_SOA_COLUMN(P, p, float);
 DECLARE_SOA_COLUMN(Sign, sign, float);
+DECLARE_SOA_COLUMN(Y, y, float);
```

Comment:
in general `Y` as a name might be a bit confusing, consider using e.g. rapidity or something else